packages/cevitxe-storage-mongodb/jest.config.js

```diff
@@ -0,0 +1,9 @@
+﻿const ts_preset = require('ts-jest/jest-preset')
+const mongo_preset = require('@shelf/jest-mongodb/jest-preset')
+const R = require('ramda')
+const presets = R.mergeRight(ts_preset, mongo_preset)
+
+module.exports = {
+  ...presets,
+  testMatch: ['**/src/*.test.ts'],
+}
```

Comment:
We need a separate config file for jest in order to merge two presets - `ts-jest` and `jest-mongodb`.